ui/app/styles/core/icon.scss

```diff
@@ -3,57 +3,6 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-$icon-dimensions: 1.25rem;
-$icon-dimensions-small: 1rem;
-$icon-dimensions-medium: 1.5rem;
-$icon-dimensions-large: 2rem;
-
-.icon {
-  align-items: center;
-  display: inline-flex;
-  justify-content: center;
+ .hds-icon {
   vertical-align: text-top;
```

Comment:
This is the only style I found myself reaching for! Without it, all the icons looked ~2 pixels too high in row-based situations.

Example below; first row has the vertical-align: text-top; rule applied, second row has it unset.
<img width="484" alt="image" src="https://github.com/user-attachments/assets/321b6132-cf63-4e41-a13d-bb11c3bed368" />